onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -1220,30 +1218,21 @@ TEST(CApiTest, test_custom_op_library) {
        17, 18, 18, 18, 17,
        17, 17, 17, 17, 17};
 
-  std::string lib_name;
+  onnxruntime::PathString lib_name;
 #if defined(_WIN32)
-  lib_name = "custom_op_library.dll";
+  lib_name = ORT_TSTR("custom_op_library.dll");
```

Comment:
>("custom_op_library.dll"); [](http://example.com/codeflow?start=21&length=26)

I tried using the name without extension on Windows, it works. I am wondering if it is true on other platforms, then it would simplify the code.

Reply:
I looked at the dlopen documentation for linux and wrote a small test program. Unfortunately, linux requires the extension. I'm not sure about other platforms, but it looks like we have to keep this code as is.